Minio.Examples/Cases/PutObject.cs

```diff
@@ -38,27 +38,28 @@ class PutObject
                 byte[] bs = File.ReadAllBytes(fileName);
                 using (MemoryStream filestream = new MemoryStream(bs))
                 {
-                    if (filestream.Length < (5 * MB))
+                    FileInfo fileInfo = new FileInfo(fileName);
+                    if (fileInfo.Length < (5 * MB))
```

Comment:
not sure why this check is even needed in this example - whether multipart upload or single upload is done should be transparent to client.